python/pyspark/sql/tests/pandas/test_pandas_transform_with_state.py

```diff
@@ -358,7 +369,7 @@ def check_results(batch_df, batch_id):
                 for q in self.spark.streams.active:
                     q.stop()
             if batch_id == 0 or batch_id == 1:
-                time.sleep(6)
+                time.sleep(4)
```

Comment:
nit: Is it relevant change? I am happy with reducing test duration, just to see the rationale.

Reply:
This is an intentional change because the CI test won't pass with prior 6 seconds. I assume this is because the pre-init stage is now added and it take a bit more time to finish the first batch so some TTL state already timed out before we get to check results on the second batch.